test/vanilla/version-tolerant/Expected/AcceptanceTests/BodyFormDataVersionTolerant/bodyformdataversiontolerant/aio/operations/__init__.py

```diff
@@ -12,6 +12,24 @@
 from ._patch import *  # type: ignore # pylint: disable=unused-wildcard-import
 from ._patch import patch_sdk as _patch_sdk
 
+
+def raise_not_implemented(target):
+    _all_abstract_methods = [
+        "FormdataOperations.upload_file",
+        "FormdataOperations.upload_files",
+    ]
+    raise NotImplementedError(
+        f"{target} Not Implemented! "
+        f"We won't initialize if any of these methods not Implemented by customization: {_all_abstract_methods} "
+        "Please refer to https://aka.ms/azsdk/python/dpcodegen/python/customize to learn how to customize."
+    )
+
+
+if not callable(getattr(FormdataOperations, "upload_file", None)):
```

Comment:
I like the list, I think we can leverage the list to get rid of the duplicate checks. i.e.

```python
abstract_methods = [
    "upload_file", "upload_files"
]
not_implemented = [f for f in abstract_methods if not callable(getattr(FormdataOperations, f))]
if any(not_implemented):
    raise NotImplementedError(f"The following methods on operation group 'FormdataOperations' are not implemented: '{'\', \''.join(not_implemented)}'")
```

Reply:
We can also vendor `raise_not_implemented` in the `vendor` file (we don't need multiple definitions for each operation group). Then, it can take in the list of abstract methods, and the operation group as parameters